### ISSUE TITLE
train image_segmentation_unet with Dice_coef_ Loss 

### ISSUE THREAD

**Issue by Defneh** [ovh/ai-training-examples#111]
Knowing that I am training using the 4 MRI modalities, when I use categorical cross-entropy,  in your tutorial from ([brain_tumor_segmentation_u_net.ipynb](https://github.com/ovh/ai-training-examples/blob/main/notebooks/computer-vision/image-segmentation/tensorflow/brain-tumor-segmentation-unet/notebook_image_segmentation_unet.ipynb),) the IOU and Dice coefficients work fine. However, when I switch to Dice loss or other loss functions, the loss doesn't change and showed same metrics in all epochs. 
Could you please help me figure out why?

When I print the values, I get the following:

y_batch min/max: 0.0 / 1.0
Model output (y_pred) min/max: 0.08728714 / 0.43650725
Could this discrepancy be the cause of the issue?
Thank you so much for your help
